dev/TOC.md

```diff
@@ -349,6 +349,7 @@
       - [监控告警](/dev/reference/tools/tidb-lightning/monitor.md)
       - [故障诊断](/dev/how-to/troubleshoot/tidb-lightning.md)
       - [FAQ](/dev/faq/tidb-lightning.md)
+      - [TiDB Lightning 术语表](/dev/reference/tools/tidb-lightning/glossary.md)
```

Comment:
```suggestion
      - [术语表](/dev/reference/tools/tidb-lightning/glossary.md)
```

I don't think we need to repeat "TiDB Lightning" here since the level above is already "TiDB Lightning".